src/dotnet/APIView/APIViewWeb/Models/AIReviewModel.cs

```diff
@@ -10,21 +10,8 @@ namespace APIViewWeb.Models
 
     public class AIReviewComment
     {
-        private List<string> _guidelineIds;
-
-        [JsonPropertyName("rule_ids")]
-        public List<string> RuleIds 
-        { 
-            get => _guidelineIds; 
-            set => _guidelineIds = value; 
-        }
-        
-        [JsonPropertyName("guideline_ids")]
-        public List<string> GuidelineIds 
-        { 
-            get => _guidelineIds; 
-            set => _guidelineIds = value; 
-        }
+        [JsonPropertyName("guideline_ids")] 
+        public List<string> GuidelineIds { get; set; }
```

Comment:
Don't we have comments in the DB that are still stored with `rule_ids`?

Reply:
Ah, how were they stored before? Did we just stuff the guideline link into the text of the comment?

Reply:
yes, it was just part of the comment :)